src/sentry/event_manager.py

```diff
@@ -1120,11 +1120,7 @@ def discard_event(job, attachments):
     metrics.incr(
         "events.discarded",
         skip_internal=True,
-        tags={
-            "organization_id": project.organization_id,
-            "platform": job["platform"],
-            "project_id": job["project_id"],
-        },
+        tags={"organization_id": project.organization_id, "platform": job["platform"]},
```

Comment:
```suggestion
        tags={"platform": job["platform"]},
```

Let's also remove this while we're on it.